### DISCUSSION TITLE
Gathering SQL configuration Q.

### DISCUSSION THREAD

**Discussion by Schutzhund3** [dataplat/dbatools#9356] [Q&A]
Would I be able to get all the SQL servers' configuration (including mirroring, clustering, high availability etc.) and also (as a second Q), could I also get all of a database configuration (schema, column types etc.) with DBATools?

**Comment by wsmelton** [comment]
All of the content mentioned is available through various methods whether it be one single-command (e.g., [Get-DbaInstanceProperty](https://docs.dbatools.io/Get-DbaInstanceProperty)) or mixing commands that pull the details you need.

You may find blog posts online of the user community that have written wrapper scripts to collect all the data.
